ci-operator/config/maistra/istio/maistra-istio-maistra-3.0.yaml

```
@@ -69,6 +69,63 @@ tests:
           memory: 100Mi
       timeout: 2h0m0s
     workflow: servicemesh-istio-e2e-aws
+- as: maistra-istio-unit-3-0
+  commands: |
+    make -e T=-v build racetest binaries-test \
+    BUILD_WITH_CONTAINER="0" \
+    GOBIN="/gobin" \
+    GOCACHE="/tmp/cache" \
+    GOMODCACHE="/tmp/cache" \
+    XDG_CACHE_HOME="/tmp/cache"
```

```suggestion
    XDG_CACHE_HOME="/tmp/cache" \
    ARTIFACTS="${ARTIFACT_DIR:-}"
```

For unit test job, we need another env variable for artifacts directory.
Reference: https://docs.ci.openshift.org/docs/how-tos/artifacts/#ci-operator-artifacts